felix/wireguard/wireguard.go

```diff
@@ -661,7 +664,7 @@ func (w *Wireguard) Apply() (err error) {
 	}
 
 	if w.wireguardNotSupported {
-		w.logCtx.Info("Wireguard is not supported")
+		w.rateLimitedLogger.WithFields(w.logCtx.Data).Info("Wireguard is not supported")
```

Comment:
Was thinking it would be better to the `WithFields` on line 287 so that any future uses of this logger get them automatically.

Reply:
But this is plenty good as it is

Reply:
Oh right, sorry! I'll post a follow-up PR with it

Reply:
https://github.com/projectcalico/calico/pull/6518

sorry once again about the mix up